scrapper.py

```diff
@@ -1,6 +1,20 @@
 """
 Scrapper implementation
 """
+from datetime import datetime
+import json
+from pathlib import Path
+import random
+import shutil
+from time import sleep
+
+from bs4 import BeautifulSoup
+import requests
+
+from core_utils.article import Article
+from constants import ASSETS_PATH, CRAWLER_CONFIG_PATH
+
+HTTP_PATTERN = 'https://snob.ru'
```

Comment:
it is better to import this from constants

Reply:
ok.